add instructions for running performance tests

aelkiss opened the pull request:
- instructions for binary resource test
- fix typo in property name for file sizes

aelkiss committed:
add instructions for running performance tests

  - instructions for binary resource test
  - fix typo in property name for file sizes

escowles commented on README.md:
```diff
@@ -18,3 +18,52 @@ The following is an example running a test against a remote Fedora deployed unde
+
+To run with a maximum file size of 500MB:
+
+* Toggle thread groups so only 'Fedora4 Create Binary Resource' is enabled (can only be done via the GUI, but persists after saving)
```
I found enabling/disabling thread groups in the GUI was reflected pretty directly in the `fedora.jmx` file, e.g.:

```
160c160
<       <ThreadGroup guiclass="ThreadGroupGui" testclass="ThreadGroup" testname="Fedora4 Create New Containers" enabled="false">
---
>       <ThreadGroup guiclass="ThreadGroupGui" testclass="ThreadGroup" testname="Fedora4 Create New Containers" enabled="true">
```

Can we use variables to enable/disable ThreadGroups with command-line parameters?  That would make it easier for people to run the different tests.

awoods replied:
:+1: to toggling ThreadGroups via CLI parameters.

aelkiss commented:
This article has some suggestions on how to do this: http://www.testautomationguru.com/jmeter-manage-test-plan/ - essentially one needs to set the number of threads for thread groups you don't want to run to zero. I can see about adding appropriate properties to do that later this week.

escowles commented:
The size parameters for tests 1 and 2 don't seem to be working.  Regardless of the parameters I use, I'm seeing 0-byte files created in all cases.

**Update**: ignore this comment -- looks like my testing script was using the wrong version of the `fedora.jmx` file.  Now that I'm using the correct test script, I'm seeing file sizes in the specified range.  So I'll restart tests 1 and 2 on Princeton's VMs, and hopefully have some results for Monday's meeting.

aelkiss committed:
Allow choosing thread group from command line

  - Add BINARY_THREADS and CONTAINER_THREADS parameter
  - Remove unused user defined variables
  - Remove unused 'Fedora 4' thread group
  - Enable performance requests

aelkiss committed:
README for thread group command line parameters

aelkiss commented:
Sorry I didn't get to this last week; PR is updated just in time for the call!
